### DISCUSSION TITLE
add suport for FTP folders

### DISCUSSION THREAD

**Discussion by Eljoseto** [vladmandic/sdnext#1940] [Ideas]
I have a lot of storage space on a second PC that I use as a local server. It ocurred to me that I could offload the models folder to that server and connect through FTP to access the models in that disk. but FTP address doesn't work on the system paths.

It would be useful to be able to access network shared folders containing models, specially since SD.Next has a stream loading method.

**Comment by vladmandic** [comment]
FTP protocol does not have concepts of file mapping, it's just a transport protocol.

But you can use your network storage using any file mapping protocol such as CIFS/SMB or NFS or many others.

And yes, that's exactly where stream loading helps a lot.

**Comment by Eljoseto** [comment]
Alright thanks for the clarification!
